charts/skypilot/values.yaml

```diff
@@ -0,0 +1,92 @@
+apiService:
+  cpuCoresLimit: null
+  memoryGBLimit: null
```

Comment:
Specifying these to actual numbers causes issue. @aylei  is looking into this in #4731 

We should leave a comment here.

Reply:
I'm actually curious why we don't have problems without setting limits. The server just gets the host resources as limits in such cases with no other difference. Maybe it is simply because the host resource is happen to be enough for our smoke test.

Reply:
^ Could it be related to the kubelet oom killing the API server pod under resource pressure?

Reply:
Added a comment